app/controllers/ViewTransfersController.scala

```diff
@@ -13,8 +13,36 @@ import java.time.format.DateTimeFormatter
 import java.util.UUID
 import javax.inject.Inject
 
+// deliberately not using .get for Maps so that it throws an exception as we should be alerted if values in DB have changed
 class ViewTransfersController @Inject() (val consignmentService: ConsignmentService, val keycloakConfiguration: KeycloakConfiguration, val controllerComponents: SecurityComponents)
     extends TokenSecurity {
+
+  private val inProgressStepStatus = "InProgress"
+  private val failedStepStatus = "Failed"
+  private val completedWithIssuesStepStatus = "CompletedWithIssues"
+  private val completedStepStatus = "Completed"
+
+  private val inProgressConsignment = "In Progress"
```

Comment:
I would rename these like this `transferStatusInProgress` etc, then it matches with the name of the field in the `UserAction` case class